Assets/Sanji.cs

```diff
@@ -13,6 +14,8 @@ public class Sanji : Character {
 
 		player.knockbackVector = new Vector2 (direction > 0 ? -1 : 1, 1);
 		player.wasThrown (direction);
+
+		hitter.GetComponent<Health> ().Damage (kickDmg);
```

Comment:
should be
```
collider.GetComponent<Health> ().Damage (kickDmg);
```
Perhaps I should rename those to make more sense, the hitter is the one who successfully hit, the collider is the one who got hit (and takes the damage and knockback vector).